24-generics/01-sorting/.README.md

```diff
@@ -0,0 +1,8 @@
+# Sorting data
+
+Write a function that receives a slice as its argument and returns a slice that contains the original values in ascending order (if the slice contains strings, sort from 'A' - 'Z').
+The sorting should satisfy the following requirements:
+- The function should work with slices of type *`{{ index . "type-one"}} and {{index . "type-two"}} }`* but is not required to work with other types
+- You must not use type checking and the function must not accept a slice with type interface{} (hint: read the name of the parent directory :D)
+//TODO példa
```

Comment:
Please add the missing example, I'm afraid people will not understand without it. The equivalent Python would be perfect, as in the other exercise.

Reply:
You can maybe add a separate example in the yaml def and then just show the right example here based on the index?